config/routes.rb

```diff
@@ -7,6 +7,9 @@
   # end
   
   root 'welcome#home'
+  get 'welcome/about'
+  get 'welcome/research'
+  get 'welcome/contact'
```

Comment:
@ianroberts131 In case you wanted shorter URL paths for these pages:

```rb
get 'about' => 'welcome#about' # URL will be https://host/about
get 'research' => 'welcome#research' # URL will be https://host/research
```